ParabolInc/parabol
# Spotlight — highlight search keyword

#5545

https://user-images.githubusercontent.com/466991/143566195-bcefd471-f8ab-4490-b90f-1844ea13f4b4.mov

**How to test**

- Open spotlight
- Type anything in a search field
- See matched keyword highlighted
- If the matched reflection card is deep inside a reflection group, that card should pop up on the top at the stack
